game-core/src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -127,7 +125,7 @@
   private String defenderOrderOfLosses = null;
   private Territory location = null;
   private JList<String> territoryEffectsJList;
-  private final WidgetChangedListener listenerPlayerUnitsPanel = () -> setWidgetActivation();
+  private final WidgetChangedListener listenerPlayerUnitsPanel = this::setWidgetActivation;
```

Comment:
Is there a reason why this field cannot be inlined below?

```java
attackingUnitsPanel.addChangeListener(this::setWidgetActivation);
defendingUnitsPanel.addChangeListener(this::setWidgetActivation);
```

If it's needed for some reason, could you please rename it so the `listener` prefix is moved to the suffix instead (i.e. `playerUnitsPanelListener`)?  The `Panel` suffix makes it seem like this is a UI component.

Reply:
(side note) Ideally rename too : )
`widgetActivation` is a bit too generic to have real meaning.

Reply:
@DanVanAtta Any suggestions for a fitting name?